osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -869,6 +895,7 @@ private class CarouselScrollContainer : OsuScrollContainer
             /// </summary>
             public bool UserScrolling { get; private set; }
 
+            // ReSharper disable once OptionalParameterHierarchyMismatch fuck off rider
```

Comment:
```suggestion
            // ReSharper disable once OptionalParameterHierarchyMismatch
```
At least this, if not removing. (it doesn't fail pre-EAP fwiw).

Reply:
oops, wasn't supposed to be committed